test/functional/inspec_exec_json_test.rb

```diff
@@ -134,11 +137,14 @@
       _(ex3["impact"]).must_equal 0.9
     end
 
+    make_my_diffs_pretty!
+
```

Comment:
Prolly should remove... only needed while debugging. Slower otherwise (tho... only on failures).